config.lua

```diff
@@ -11,7 +11,7 @@ if ox.server then
 	Config.CheckVersion = false
 
 	-- When should unused inventories be wiped
-	Config.DBCleanup = '6 MONTH'
+	Config.DBCleanup = '6' -- Months
```

Comment:
What if user wants to clean it up after `X DAY` ? Also it is more readable when there are month inside config imo.